demos/projects/ESPRESSIF/esp32/components/azure-iot-middleware-freertos/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+set(ROOT_PATH
```

Comment:
please add 

# Copyright (c) Microsoft Corporation. All rights reserved.
# SPDX-License-Identifier: MIT

to your cmakes

Reply:
Done.